src/algorithms/calorimetry/CalorimeterHitDigi.h

```diff
@@ -13,6 +13,7 @@
 
 #pragma once
 
+#include <gsl/gsl>
```

Comment:
Unused? (and ditto for other occurrences)

Reply:
Yeah, didn't include that in my sed. Let me add (subtract) them.